schema/url/correlationassaymetadata/plugin-correlationassaymetadata-bipartitenetwork.raml

```diff
@@ -6,8 +6,22 @@ types:
   CorrelationAssayMetadataBipartitenetworkPostRequest:
     type: DataPluginRequestBase
     properties:
-      computeConfig: EmptyDataPluginSpec
-      config: EmptyDataPluginSpec
+      computeConfig: CorrelationComputeConfig
+      config: CorrelationAssayMetadataBipartitenetworkSpec
 
   CorrelationAssayMetadataStatsResponse:
-    type: DifferentialAbundanceStatsResponse
+    additionalProperties: false
+    properties:
+      statistics: CorrelationPoint[]
+      data1Metadata: string
+      data2Metadata: string
+
+  CorrelationAssayMetadataBipartitenetworkSpec:
```

Comment:
similar question.. how is this specific to assay-metadata?

Reply:
updated